src/main/java/com/hazelcast/cloud/JdbcClient.java

```diff
@@ -0,0 +1,66 @@
+package com.hazelcast.cloud;
+
+import com.hazelcast.client.HazelcastClient;
+import com.hazelcast.client.config.ClientConfig;
+import com.hazelcast.cloud.model.City;
+import com.hazelcast.config.SSLConfig;
+import com.hazelcast.core.HazelcastInstance;
+
+import java.net.URISyntaxException;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.Statement;
+import java.util.Properties;
+
+import static com.hazelcast.cloud.ClientWithSsl.*;
+
+public class JdbcClient {
+
+    public static void main(String[] args) throws URISyntaxException {
+        // Configure the client
+        ClassLoader classLoader = JdbcClient.class.getClassLoader();
+        Properties tlsProperties = new Properties();
+        tlsProperties.setProperty("javax.net.ssl.keyStore", classLoader.getResource("client.keystore").toURI().getPath());
+        tlsProperties.setProperty("javax.net.ssl.keyStorePassword", "YOUR_SSL_PASSWORD");
+        tlsProperties.setProperty("javax.net.ssl.trustStore",
+                classLoader.getResource("client.truststore").toURI().getPath());
+        tlsProperties.setProperty("javax.net.ssl.trustStorePassword", "YOUR_SSL_PASSWORD");
```

Comment:
did we break these out into tokens?

Reply:
I am unaware, this is how it is used everywhere

Reply:
Normally, when you download the sample from the cloud, these `YOUR_XXX` parts are automatically replaced with the actual values. So, when you download the sample, you are all set, and it is ready to run. When we put these into a new file like this, is the cloud still replacing the values in this file?

Reply:
@mdumandag Not at the moment, but it will be addressed by a separate PR once this PR is approved (I will send you a link)

Reply:
The alternative is to keep the client in `ClientWithSsl`, but Avtar prefers we keep them separate